packages/credential-ld/src/ld-credential-module.ts

```diff
@@ -46,16 +50,20 @@ export class LdCredentialModule {
 
         if (!didDoc) return
 
+        let result: any = didDoc
```

Comment:
this needs to be `any`?

Reply:
Yeah, because later it can become a verification method instead of a did doc